VideoLocker/src/main/java/org/edx/mobile/view/NavigationFragment.java

```diff
@@ -112,17 +113,30 @@ private void loadProfileImage(@NonNull ProfileImage profileImage, @NonNull Image
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
-        View layout = inflater.inflate(R.layout.drawer_navigation, container, false);
+        final View layout = inflater.inflate(R.layout.drawer_navigation, container, false);
 
-        TextView name_tv = (TextView) layout.findViewById(R.id.name_tv);
-        TextView email_tv = (TextView) layout.findViewById(R.id.email_tv);
+        final TextView name_tv = (TextView) layout.findViewById(R.id.name_tv);
+        final TextView email_tv = (TextView) layout.findViewById(R.id.email_tv);
+        final FrameLayout nameLayout = (FrameLayout)layout.findViewById(R.id.name_layout);
```

Comment:
> Should these views be injected instead?

Why? Doing it this way is better understood by the IDE and Linter and is not any more difficult.